debug/examples/example-files.json

```diff
@@ -26,6 +26,11 @@
                 "type": "json5",
                 "key": "background-animations",
                 "url": "background-animations.json5"
+            },
+            {
+                "type": "json5",
+                "key": "background-particles",
```

Comment:
Is this going to be loaded as a particles file in future?

Reply:
Oh wait sorry, this is a screen isn’t it 😂

Reply:
Ha! 
Yeah I actually debated renaming those to furniture but I think they make more sense as is.